mmv1/templates/terraform/decoders/spanner_database.go.erb

```diff
@@ -6,6 +6,9 @@ if err := tpgresource.ParseImportId([]string{"projects/(?P<project>[^/]+)/instan
 res["project"] = d.Get("project").(string)
 res["instance"] = d.Get("instance").(string)
 res["name"] = d.Get("name").(string)
+if _, ok := res["enableDropProtection"]; !ok {
+	res["enableDropProtection"] = false
+}
```

Comment:
Can you investigate that if this is needed for the decoder? I suspect we can remove it. Sorry that I missed it in my last review.